deploy/bundle/manifests/devworkspace-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,373 @@
+          - apps
+          - extensions
+          resources:
+          - replicasets
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - batch
+          resources:
+          - jobs
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - controller.devfile.io
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        - apiGroups:
+          - controller.devfile.io
+          resources:
+          - devworkspaceroutings
+          verbs:
+          - '*'
+        - apiGroups:
+          - controller.devfile.io
+          resources:
+          - devworkspaceroutings/status
+          verbs:
+          - get
+          - patch
+          - update
+        - apiGroups:
+          - extensions
+          resources:
+          - ingresses
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - create
+          - get
+        - apiGroups:
+          - oauth.openshift.io
+          resources:
+          - oauthclients
+          verbs:
+          - create
+          - delete
+          - deletecollection
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterrolebindings
+          - clusterroles
+          - rolebindings
+          - roles
+          verbs:
+          - create
+          - get
+          - list
+          - update
+          - watch
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          verbs:
+          - '*'
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes/custom-host
+          verbs:
+          - create
+        - apiGroups:
+          - workspace.devfile.io
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: devworkspace-controller-serviceaccount
+      deployments:
+      - name: devworkspace-controller-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app.kubernetes.io/name: devworkspace-controller
+              app.kubernetes.io/part-of: devworkspace-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                app.kubernetes.io/name: devworkspace-controller
+                app.kubernetes.io/part-of: devworkspace-operator
+            spec:
+              containers:
+              - args:
+                - /usr/local/bin/devworkspace-controller
+                - --enable-leader-election
+                env:
+                - name: DEVELOPMENT_MODE
+                  value: "true"
```

Comment:
I know the CSV isn't fully done, but should this be false for production? Or would it be easier to diagnose issues if we have more log info?

Reply:
It absolutely should be -- now to figure out where this value is coming from

Reply:
Turns out the easiest way to fix this was to just disable the `manager_dev_mode_patch.yaml` kustomization. We should eventually figure out a way to enable it (I think `export DEVELOPMENT_MODE=truel; make run` should work...)